src/main/java/seedu/tasklist/model/task/Time.java

```diff
@@ -21,22 +21,22 @@
             .withResolverStyle(ResolverStyle.STRICT);
 
     /**
-     * Constructs a {@code Deadline}.
+     * Constructs a {@code Time}.
      *
      * @param deadline A valid deadline.
      */
-    public Deadline(String deadline) {
+    public Time(String deadline) {
         requireNonNull(deadline);
```

Comment:
If I am not mistaken, the time constructor is used for both assignments and lessons. Hence, it may be more apt to rename the parameter of the constructor as well :)